sjb/config/test_cases/test_pull_request_origin_aggregated_logging_journald.yml

```diff
@@ -0,0 +1,71 @@
+---
+parent: 'common/test_cases/origin_built_installed_release.yml'
+overrides:
+  sync:
+    - "openshift,aos-cd-jobs=master"
+    - "openshift,openshift-ansible=master"
+  evars: "-e skip_sanity_checks=true"
```

Comment:
need to tell origin install to use journald - add `-e openshift_docker_log_driver=journald -e openshift_docker_options=--log-driver=journald`
likewise for the json-file jobs - add `-e openshift_docker_log_driver=json-file -e openshift_docker_options=--log-driver=json-file`